website/content/en/status/report-2023-04-2023-06/batman.adoc

```diff
@@ -0,0 +1,21 @@
+=== BATMAN support in the FreeBSD kernel
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/CallingTheBatmanFreeNetworksOnFreeBSD[Wiki page] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/CallingTheBatmanFreeNetworksOnFreeBSD[] +
+link:https://github.com/obiwac/freebsd-gsoc/pull/1[Source code (PR)] URL: https://github.com/obiwac/freebsd-gsoc/pull/1[]
```

Comment:
Please avoid using the PR abbreviation: we use it in different contexts with different meanings (pull requests or problem report).
```suggestion
link:https://github.com/obiwac/freebsd-gsoc/pull/1[Source code (Pull Request)] URL: https://github.com/obiwac/freebsd-gsoc/pull/1[]
```